src/Sarif.Multitool/RewriteCommand.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using Microsoft.CodeAnalysis.Sarif.Readers;
+using Microsoft.CodeAnalysis.Sarif.Visitors;
+using Microsoft.CodeAnalysis.Sarif.Writers;
+using Newtonsoft.Json;
+
+namespace Microsoft.CodeAnalysis.Sarif
+{
+    internal static class RewriteCommand
+    {
+        public static int Run(RewriteOptions rewriteOptions)
+        {
+            try
+            {
+                rewriteOptions = ValidateOptions(rewriteOptions);
+                string fileName = GetOutputFileName(rewriteOptions);
+
+                Formatting formatting = rewriteOptions.PrettyPrint ? Formatting.Indented : Formatting.None;
+
+                JsonSerializerSettings settings = new JsonSerializerSettings()
+                {
+                    ContractResolver = SarifContractResolver.Instance,
+                    Formatting = Formatting.Indented
+                };
+
+                string sarifText = File.ReadAllText(rewriteOptions.InputFilePath);
+                SarifLog actualLog = JsonConvert.DeserializeObject<SarifLog>(sarifText, settings);
```

Comment:
>actualLog [](start = 25, length = 9)

Consider: `inputLog`.

I usually reserve names `actualFoo` for unit tests, where I compare them to `expectedFoo`.